### PR TITLE
test(integration): do not use hardcoded artifact name

### PR DESCRIPTION
**Pull request by kjaskiewiczz**

### CODE REVIEW

File: tests/tests/test_basic_integration.py [hunk 1]
```diff
@@ -73,7 +75,9 @@ def do_test_double_update_rofs(self, env, valid_image_rofs_with_mender_conf):
         # see: https://github.com/mendersoftware/mender-qa/pull/580/commits/a4c54e9ebf5879f0837789cbc7c74cb1102b2caa#diff-7031fcd5c934f225781febe446a34594a9a36b335c540379c0dd376a8f06ba28R263
         # so first we expect it to be set to master, as this is what we
         # have in the initial image, after update we expect it to change
-        expected_artifact_info = "artifact_name=mender-image-master\n"
+        expected_artifact_info = (
+            "artifact_name=mender-image-" + mender_client_version + "\n"
```
Comment (oleorhagen) [inline tests/tests/test_basic_integration.py:79]: Suggestion:
`f"artifact_name=mender-image{mender_client_version}\n"

File: tests/conftest.py [hunk 2]
```diff
@@ -275,6 +275,16 @@ def min_mender_client_version(request):
...
 
 
+@pytest.fixture(autouse=True)
+def mender_client_version():
+    mender_client_version = (
+        subprocess.check_output([RELEASE_TOOL, "--version-of", "mender"])
+        .decode()
+        .strip()
+    )
+    return mender_client_version
```
Comment (oleorhagen) [inline tests/conftest.py:285]: Suggestion: 

How about just returning the statement straight up, we already know from the function name what's going on?

**Review by oleorhagen — APPROVED** [review]
:tada:

File: tests/conftest.py (hunk 2 above)
Comment (kjaskiewiczz) [reply tests/conftest.py]: done

**Comment by oleorhagen** [comment]
@mender-test-bot start pipeline

**Pull request merged**